app-emulation/img/img-0.5.7-r1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+EGO_PN="github.com/genuinetools/img"
+
+inherit golang-vcs-snapshot
+
+DESCRIPTION="Standalone daemon-less unprivileged Dockerfile and OCI container image builder"
+HOMEPAGE="https://github.com/genuinetools/img"
+SRC_URI="https://github.com/genuinetools/img/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64"
+LICENSE="MIT"
```

Comment:
Remember that these packages need to tell licenses of bundled deps as well. You can use `go-license-detector` to get a grasp what those might be for this package.

Reply:
/tmp/portage/app-emulation/img-0.5.7-r1/work/img-0.5.7/src/github.com/genuinetools/img
```
	99%	MIT
	84%	MIT-feh
```